Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -2162,20 +2196,42 @@ void TextureCacheBase::CopyRenderTargetToTexture(
                           isIntensity, gamma, clamp_top, clamp_bottom,
                           GetVRAMCopyFilterCoefficients(filter_coefficients));
 
-      if (g_ActiveConfig.bDumpEFBTarget && !is_xfb_copy)
+      if (is_xfb_copy)
       {
-        static int efb_count = 0;
-        entry->texture->Save(
-            fmt::format("{}efb_frame_{}.png", File::GetUserPath(D_DUMPTEXTURES_IDX), efb_count++),
-            0);
-      }
+        const std::string_view name =
+            fmt::format("{}_{}x{}_{}", xfb_dump_prefix, tex_w, tex_h, static_cast<int>(baseFormat));
+        if (g_ActiveConfig.bGraphicMods)
+        {
+          entry->texture_info_name = name;
+        }
 
-      if (g_ActiveConfig.bDumpXFBTarget && is_xfb_copy)
+        if (g_ActiveConfig.bDumpXFBTarget)
+        {
+          static int xfb_count = 0;
+          entry->texture->Save(
+              fmt::format("{}{}_c{}.png", File::GetUserPath(D_DUMPTEXTURES_IDX), name, xfb_count++),
+              0);
+        }
+      }
+      else
       {
-        static int xfb_count = 0;
-        entry->texture->Save(
-            fmt::format("{}xfb_copy_{}.png", File::GetUserPath(D_DUMPTEXTURES_IDX), xfb_count++),
-            0);
+        if (g_ActiveConfig.bDumpEFBTarget || g_ActiveConfig.bGraphicMods)
+        {
+          const std::string_view name = fmt::format("{}_{}x{}_{}", efb_dump_prefix, tex_w, tex_h,
+                                                    static_cast<int>(baseFormat));
```

Comment:
Uh, I don't think this is safe? `fmt::format()` produces a `std::string` that you immediately take a `std::string_view` from and then discard. `std::string_view` requires the backing string to still exist during its lifetime.